assets/js/gathercontent.js

```diff
@@ -79,6 +79,18 @@ window.GatherContent = window.GatherContent || {};
 		}
 	};
 
+	main.maybeRevealComponent = function (evt) {
+		var $this = $(this);
+		evt.preventDefault();
+		console.log('maybeRevealComponent');
```

Comment:
Are we able to clean up all the console.log? These feel very much development tool and really shouldn't be part of the production plugin.